src/node_util.cc

```diff
@@ -91,11 +91,12 @@ static void GetProxyDetails(const FunctionCallbackInfo<Value>& args) {
   if (!args[0]->IsProxy())
     return;
 
-  CHECK(args[1]->IsBoolean());
-
   Local<Proxy> proxy = args[0].As<Proxy>();
 
-  if (args[1]->IsTrue()) {
+  // TODO(BridgeAR): Remove the length check as soon as we prohibit access to
+  // the util binding layer. It's accessed in the wild and `esm` would break in
+  // case the check is removed.
+  if (args.Length() == 1 || args[1]->IsTrue()) {
```

Comment:
The length check here doesn’t change behaviour and could safely be dropped, so you can apply the `TODO` comment right now if you want.

Reply:
You mean if I write `if (!args[1]->IsFalse()) { ...old behavior... } else { ...new behavior... }`?

Reply:
Yes, something like that … feel free to ignore though :)